benchmark/es/map-bench.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const common = require('../common.js');
+const assert = require('assert');
+
+const bench = common.createBenchmark(main, {
+  method: ['pojo', 'fakeMap', 'map'],
```

Comment:
plain old javascript object ... comes from plain old java object, maybe that's just for us old java dogs